mne/preprocessing/maxwell.py

```diff
@@ -31,6 +31,8 @@ def maxwell_filter(raw, origin=(0, 0, 40), int_order=8, ext_order=3,
         Order of internal component of spherical expansion
     ext_order : int
         Order of external component of spherical expansion
+    tSSS_buffer : float
```

Comment:
the capital letters bother me too, how about `st_duration` or `st_dur`? I think the param in `MaxFilter` is actually `st` so it resonates there, too.